packages/web3-wrapper/src/marshaller.ts

```diff
@@ -25,7 +25,14 @@ import {
     TxDataRPC,
 } from './types';
 
+/**
+ * Utils to convert ethereum structures from user-space format to RPC format. (marshall/unmarshall)
+ */
 export const marshaller = {
```

Comment:
Can you also import the marshaller from the public interface in SignerSubprovider now that it's exported?